app/middleware/csp.js

```diff
@@ -0,0 +1,19 @@
+const helmet = require('helmet')
+
+const applyCspRulesEnabled = process.env.CSP_SEND_HEADER === 'true'
```

Comment:
`applyCspRulesEnabled` could just be `sendCspHeader` or something to more closely match the environment variable name